{
  "name": "throw-http-errors",
  "version": "3.2.0",
  "description": "Custom errors for HTTP status codes.",
  "main": "lib/index.js",
  "scripts": {
    "build": "tsc",
    "prepublish": "cp -R src lib/",
    "coverage": "istanbul cover node_modules/ts-mocha/bin/ts-mocha tests/{**/,/}*.ts -- -R spec",
    "precoveralls": "istanbul cover node_modules/ts-mocha/bin/ts-mocha tests/{**/,/}*.ts --report lcovonly -- -R spec && npm i coveralls@2",
    "coveralls": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.ts",
    "test": "make test",
    "tests": "ts-mocha tests/**/*.test.ts",
    "lint": "eslint src/**/*.ts"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/eldimious/throw-error.git"
  },
  "keywords": [
    "throw",
    "errors",
    "throw errors",
    "http errors"
  ],
  "author": "Dimos Botsaris <botsaris.d@gmail.com>",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/eldimious/throw-error/issues"
  },
  "homepage": "https://github.com/eldimious/throw-error#readme",
  "devDependencies": {
    "@types/chai": "^4.3.0",
    "@types/expect": "^24.3.0",
    "@types/mocha": "^9.0.0",
    "@types/node": "^17.0.8",
    "@typescript-eslint/eslint-plugin": "^5.9.1",
    "@typescript-eslint/parser": "^5.9.1",
    "chai": "^4.3.4",
    "coveralls": "^3.1.1",
    "eslint": "^8.6.0",
    "eslint-config-airbnb": "^19.0.4",
    "eslint-config-airbnb-typescript": "^16.1.0",
    "eslint-plugin-prettier": "^4.0.0",
    "istanbul": "^0.4.5",
    "nyc": "^15.1.0",
    "prettier": "^2.5.1",
    "ts-mocha": "^8.0.0",
    "ts-node": "^10.4.0",
    "typescript": "^4.5.4"
  }
}
